cli/utils.go

```diff
@@ -106,17 +105,21 @@ func loadAlertmanagerConfig(ctx context.Context, alertmanagerURL *url.URL, confi
 		return cfg, nil
 	}
 	if alertmanagerURL != nil {
-		status, err := getRemoteAlertmanagerConfigStatus(ctx, alertmanagerURL)
+		configStatus, err := getRemoteAlertmanagerConfigStatus(ctx, alertmanagerURL)
 		if err != nil {
 			return nil, err
 		}
-		return status.ConfigJSON, nil
+		conf, err := config.Load(*configStatus.Config.Original)
```

Comment:
it could be `return config.Load(*configStatus.Config.Original)` here.

Reply:
Done!